src/abstracts/VirtualModuleConstants.sol

```diff
@@ -0,0 +1,28 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.25;
+
+import {IRewardPoolDepositWrapper} from "../interfaces/aura/IRewardPoolDepositWrapper.sol";
+import {IBoosterLite} from "../interfaces/aura/IBoosterLite.sol";
+import {IBaseRewardPool4626} from "../interfaces/aura/IBaseRewardPool4626.sol";
+
+import "@balancer-v2/interfaces/contracts/vault/IVault.sol";
+
+abstract contract VirtualModuleConstants {
+    uint16 constant MAX_BPS = 10_000;
+
+    uint256 public constant EURE_TOKEN_BPT_INDEX = 2;
+    uint256 public constant EURE_TOKEN_BPT_INDEX_USER = 1;
+
+    address public constant MULTICALL3 = 0xcA11bde05977b3631167028862bE2a173976CA11;
+    address public immutable CARD;
```

Comment:
why leave some immutables in the contract and move some here?

Reply:
actually this a C&P mistake. i was not intentionally moving the immutable components into the abstract, there should be on its own contract instead, since they are defined on the constructor. prefer to leave into the abstract whichever is literally not dynamically set on the constructor